examples/tcp-retransmit.bpf.c

```diff
@@ -0,0 +1,84 @@
+#include <vmlinux.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+#include "string.h"
+#include "maps.bpf.h"
+
+#define MAX_ENTRIES 8192
+#define RETRANSMIT  1
```

Comment:
Empty line separation that you've had between define groups here before was nice.

Reply:
Changed.